### ISSUE TITLE
Updating the module a little?

### ISSUE THREAD

**Issue by eklem** [rexxars/vinmonopolet#8]
Hi @rexxars ,
Nice module! It seems to be a little outdated compared to how the Vinmonopolet's pages are today.
I'm not a good programmer, but would you consider updating, and I can test and create issues?

My interest is to make test-data for norch / search-index.

**Comment by eklem** [comment]
Ok, just let me know =) Also, maybe connecting to the csv-files is a better idea now? Forgot they existed: https://www.vinmonopolet.no/datadeling

**Comment by rexxars** [comment]
The module already uses the CSVs, for certain parts (`getProductStream()` and `getStoreStream()`). Vinmonopolet changed the URLs for the CSVs however. I've pushed a new version, `2.0.3` which fixes these two methods.

Working on a full rewrite of this module which will be much more reliable.

**Comment by eklem** [comment]
Also, the product links in the CSV-file are wrong, but are being rewritten. The new are on this form:
```
https://www.vinmonopolet.no/vmpSite/Land/Italia/Guerrini-Il-Terziere-Toscana-2010/p/101401
```
But it seems the real url is:
```
https://www.vinmonopolet.no/p/101401
```

**Comment by rexxars** [comment]
Just released 3.0 with a brand new API. I had to remove certain methods since I have not been able to reliably find any pattern for getting all categories, for instance.

I tried to do my best in rewriting the URLs, but there are certain exceptions that will still redirect you to some other URL. Not much to about that apart from actually doing a request, however, which would be an extreme overhead to do for every since product.

Please open issues for individual problems or bugs you find.

**Comment by eklem** [comment]
Cool. I'll start testing!
